datadog/resource_datadog_logs_custom_pipeline.go

```diff
@@ -623,6 +661,27 @@ func buildDatadogUserAgentParser(tfProcessor map[string]interface{}) datadog.Use
 	return ddUserAgentParser
 }
 
+func buildDatadogLookupProcessor(tfProcessor map[string]interface{}) datadog.LookupProcessor {
+	ddLookupProcessor := datadog.LookupProcessor{}
```

Comment:
are `name` and `is_enabled` missing here?

Reply:
`name` & `is_enabled` are handled in a generic way here https://github.com/terraform-providers/terraform-provider-datadog/blob/e0e835892b09901ae260e26bfc9d2feccd88c07c/datadog/resource_datadog_logs_custom_pipeline.go#L626-L631

Reply:
Thanks!